shoplist/src/main/java/com/netguru/codereview/ui/model/ShopList.kt

```diff
@@ -0,0 +1,11 @@
+package com.netguru.codereview.ui.model
+
+import com.netguru.codereview.network.model.ShopListItemResponse
+
+class ShopList(
+    val id: String,
```

Comment:
Is this supposed to be a shopping list item or a list of shops. Based on properties it looks like it's a list of shopping items. If later please change the name to reflect the proper item.